README.md

```diff
@@ -1,6 +1,6 @@
 # Kroxylicious
 
-Kroxylicious, the snappy open source proxy for Apache Kafka
+Kroxylicious, the snappy open source proxy for [Apache Kafka®](https://kafka.apache.org).
```

Comment:
Do we only need the ® on the first mention in the page? we have another mention of Kafka at line 5.

Reply:
First/main usage is the official ask yes.